addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -1,50 +1,55 @@
-channel-type.kodi.mute.description = Das Kodi Media Centers stumm schalten
-channel-type.kodi.control.label = Kontrolle
-channel-type.kodi.control.description = Steuert das Kodi Media Center (z.B. Play, Pause, Stop)
+channel-type.kodi.mute.description = Erm�glicht die Lautst�rke auf stumm zu schalten.
+channel-type.kodi.control.label = Fernbedienung
+channel-type.kodi.control.description = Erm�glicht die Steuerung der Wiedergabe von Medien auf dem Kodi Media Center (z.B. Play, Pause).
 channel-type.kodi.stop.label = Stop
-channel-type.kodi.stop.description = Stopt die Wiedergabe
-channel-type.kodi.playuri.label = URI abspielen
-channel-type.kodi.playuri.description = Spielt einen URI ab
-channel-type.kodi.pvr-open-tv.label = PVR TV Kanal abspielen
-channel-type.kodi.pvr-open-tv.description = Spielt einen PVR TV Kanal ab
-channel-type.kodi.pvr-open-radio.label = PVR Radio Kanal abspielen
-channel-type.kodi.pvr-open-radio.description = Spielt einen PVR Radio Kanal ab
+channel-type.kodi.stop.description = Erm�glich die Wiedergabe zu stoppen.
+channel-type.kodi.playuri.label = URI
+channel-type.kodi.playuri.description = Erm�glich das Abspielen einer URI.
+channel-type.kodi.pvr-open-tv.label = PVR TV Kanal
+channel-type.kodi.pvr-open-tv.description = Erm�glich das Abspielen eines PVR TV Kanals.
+channel-type.kodi.pvr-open-radio.label = PVR Radio Kanal
+channel-type.kodi.pvr-open-radio.description = Erm�glich das Abspielen eines PVR Radio Kanals.
 channel-type.kodi.pvr-channel.label = PVR Kanal
-channel-type.kodi.pvr-channel.description = Titel des aktuellen PVR Kanals
+channel-type.kodi.pvr-channel.description = Zeigt den Titel des aktuellen PVR Kanals an.
 channel-type.kodi.shownotification.label = Nachricht anzeigen
-channel-type.kodi.shownotification.description = Zeigt eine Nachricht auf dem Bildschirm an
+channel-type.kodi.shownotification.description = Erm�glicht die Anzeige eine Nachricht auf dem Kodi Media Center an.
 channel-type.kodi.input.label = Tastendruck
-channel-type.kodi.input.description = Schickt einen Tastendruck an das Kodi Media Center um auf dem Bildschirm zu navigieren
+channel-type.kodi.input.description = Erm�glicht das Senden eines Eingabebefehls an das Kodi Media Center.
 channel-type.kodi.inputtext.label = Texteingabe
-channel-type.kodi.inputtext.description = Schickt eine Zeichenkette als Eingabe an das Kodi Media Center
+channel-type.kodi.inputtext.description = Erm�glicht das Senden eines Eingabetext an das Kodi Media Center.
 channel-type.kodi.systemcommand.label = System Kommando
-channel-type.kodi.systemcommand.description = Schickt ein System Kommando zum Kodi Media Center um das Media Center neu zu starten, in den Ruhezustand oder Stromsparmodus zu versetzen, herunterzufahren
+channel-type.kodi.systemcommand.description = Erm�glicht das Senden eines System Kommandos um das Kodi Media Center neu zu starten, in den Ruhezustand oder Stromsparmodus zu versetzen, herunterzufahren.
```

Comment:
"-Befehls" instead of " Kommandos".
But I am not sure if it is better. 🤔 

"[..] zu versetzen oder herunterzufahren."